ExtensionEngine/tailor
# Allow signup over OIDC

This PR adds an option to allow signup for users logging in over OIDC that do not exist in the database.

Part of #632 

#### QA Note:
To test you can play around with 2 new `.env` values added:
1. `OIDC_ALLOW_SIGNUP` - determines is signup over OIDC enabled
2. `OIDC_DEFAULT_ROLE` - sets which role will users signed up over OIDC get. If left blank or set to the invalid role it will default to `USER`

**N.B.:** This PR is pointed to #689 which should be merged before. ☝️ Don't forget to change base branch after it's merged to `develop`.
